content/app/models/file-path.js

```diff
@@ -0,0 +1,90 @@
+define("content/app/models/file-path", function(require, exports, module) {
+
+	/**
+	 * Simple class used for handling of file paths.
+	 */
+	var FilePath = Class({
+
+		$statics : {
+			FOLDER_SEPARATOR : '/',
+			EXTENSION_SEPARATOR : '.',
+		},
+
+		/**
+		 * Class constructor
+		 * @param  	{String}	path 	Full file path.
+		 */
+		constructor : function(fullPath) {
+			this._fullPath = fullPath;
```

Comment:
:+1: 
I wanted to store all necessary data in just one private field, but you are right, in this way implementation is much more simpler.